### ISSUE TITLE
new model dont work

### ISSUE THREAD

**Issue by dryks** [hzy46/fast-neural-style-tensorflow#27]
I can not train the model, the pictures are processed normally, but I'm trying to train a new model. Creates a folder in the wave folder in the models folder, and the file itself is not scrpt-done.

 3.5 gigabytes of azure vps cpu

**Comment by dryks** [comment]
artisma2@artisma:~/fast-neural-style-tensorflow$ sudo python train.py -c conf/wa                             ve.yml
INFO:tensorflow:Scale of 0 disables regularizer.
2017-07-21 16:42:11.223604: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are                              available on your machine and could speed up CPU computations.
2017-07-21 16:42:11.223738: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are                              available on your machine and could speed up CPU computations.
2017-07-21 16:42:11.223786: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use AVX instructions, but these are av                             ailable on your machine and could speed up CPU computations.
2017-07-21 16:42:11.223830: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use AVX2 instructions, but these are a                             vailable on your machine and could speed up CPU computations.
2017-07-21 16:42:11.223890: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use FMA instructions, but these are av                             ailable on your machine and could speed up CPU computations.
INFO:tensorflow:Use pretrained model pretrained/vgg_16.ckpt
INFO:tensorflow:Restoring parameters from pretrained/vgg_16.ckpt
INFO:tensorflow:Target style pattern is saved to: generated/target_style_wave.jp                             g.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Loss network layers(You can define them in "content_layers" and                              "style_layers"):
INFO:tensorflow:vgg_16/conv1/conv1_1
INFO:tensorflow:vgg_16/conv1/conv1_2
INFO:tensorflow:vgg_16/pool1
INFO:tensorflow:vgg_16/conv2/conv2_1
INFO:tensorflow:vgg_16/conv2/conv2_2
INFO:tensorflow:vgg_16/pool2
INFO:tensorflow:vgg_16/conv3/conv3_1
INFO:tensorflow:vgg_16/conv3/conv3_2
INFO:tensorflow:vgg_16/conv3/conv3_3
INFO:tensorflow:vgg_16/pool3
INFO:tensorflow:vgg_16/conv4/conv4_1
INFO:tensorflow:vgg_16/conv4/conv4_2
INFO:tensorflow:vgg_16/conv4/conv4_3
INFO:tensorflow:vgg_16/pool4
INFO:tensorflow:vgg_16/conv5/conv5_1
INFO:tensorflow:vgg_16/conv5/conv5_2
INFO:tensorflow:vgg_16/conv5/conv5_3
INFO:tensorflow:vgg_16/pool5
INFO:tensorflow:vgg_16/fc6
INFO:tensorflow:vgg_16/fc7
INFO:tensorflow:vgg_16/fc8
INFO:tensorflow:Use pretrained model pretrained/vgg_16.ckpt
INFO:tensorflow:Restoring parameters from pretrained/vgg_16.ckpt
artisma2@artisma:~/fast-neural-style-tensorflow$

**Comment by dryks** [comment]
```
**XXXX@XXXX:~/fast-neural-style-tensorflow$ sudo python train.py -c conf/wave.yml                             
INFO:tensorflow:Scale of 0 disables regularizer.
2017-07-21 16:42:11.223604: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are                              available on your machine and could speed up CPU computations.
2017-07-21 16:42:11.223738: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are                              available on your machine and could speed up CPU computations.
2017-07-21 16:42:11.223786: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use AVX instructions, but these are av                             ailable on your machine and could speed up CPU computations.
2017-07-21 16:42:11.223830: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use AVX2 instructions, but these are a                             vailable on your machine and could speed up CPU computations.
2017-07-21 16:42:11.223890: W tensorflow/core/platform/cpu_feature_guard.cc:45]                              The TensorFlow library wasn't compiled to use FMA instructions, but these are av                             ailable on your machine and could speed up CPU computations.
INFO:tensorflow:Use pretrained model pretrained/vgg_16.ckpt
INFO:tensorflow:Restoring parameters from pretrained/vgg_16.ckpt
INFO:tensorflow:Target style pattern is saved to: generated/target_style_wave.jp                             g.
INFO:tensorflow:Scale of 0 disables regularizer.
INFO:tensorflow:Loss network layers(You can define them in "content_layers" and                              "style_layers"):
INFO:tensorflow:vgg_16/conv1/conv1_1
INFO:tensorflow:vgg_16/conv1/conv1_2
INFO:tensorflow:vgg_16/pool1
INFO:tensorflow:vgg_16/conv2/conv2_1
INFO:tensorflow:vgg_16/conv2/conv2_2
INFO:tensorflow:vgg_16/pool2
INFO:tensorflow:vgg_16/conv3/conv3_1
INFO:tensorflow:vgg_16/conv3/conv3_2
INFO:tensorflow:vgg_16/conv3/conv3_3
INFO:tensorflow:vgg_16/pool3
INFO:tensorflow:vgg_16/conv4/conv4_1
INFO:tensorflow:vgg_16/conv4/conv4_2
INFO:tensorflow:vgg_16/conv4/conv4_3
INFO:tensorflow:vgg_16/pool4
INFO:tensorflow:vgg_16/conv5/conv5_1
INFO:tensorflow:vgg_16/conv5/conv5_2
INFO:tensorflow:vgg_16/conv5/conv5_3
INFO:tensorflow:vgg_16/pool5
INFO:tensorflow:vgg_16/fc6
INFO:tensorflow:vgg_16/fc7
INFO:tensorflow:vgg_16/fc8
INFO:tensorflow:Use pretrained model pretrained/vgg_16.ckpt
INFO:tensorflow:Restoring parameters from pretrained/vgg_16.ckpt
XXX@XXX:~/fast-neural-style-tensorflow$**

```

**Comment by hzy46** [comment]
Check if the program successfully goes into the "while not coord.should_stop():"
(Probably killed by system because of memory issue)

**Comment by dryks** [comment]
I do not understand how to do this "while not coord.should_stop():"

**Comment by dryks** [comment]
Whether really to launch on the given script, how much did not try could not install on Windows

**Comment by dryks** [comment]
Can you train more models to run the program for example 10-20-30) on the themes of artists or other

**Comment by dryks** [comment]
How many steps in training model

**Comment by dryks** [comment]
Solution: increase the size of the VPS -7 gigabytes of RAM 2 cores  cpu
